README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
 
+### Instructions for playing the game
+
+1. Press the space bar to begin.
```

Comment:
On the pull request, click Files changed
Hover over the line of code where you'd like to add a comment, and click the blue comment icon
In the comment window, type a question or leave a more general comment
When you're done, click Add single comment